ci-operator/step-registry/openstack/ovsdpdk/machineset/openstack-ovsdpdk-machineset-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+## Enabling VFIO noiommu
+## https://docs.openshift.com/container-platform/4.9/installing/installing_openstack/installing-openstack-user-sr-iov.html#networking-osp-enabling-vfio-noiommu_installing-openstack-user-sr-iov
+cat > 05-vfio-noiommu.yaml <<EOF
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-vfio-noiommu 
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - path: /etc/modprobe.d/vfio-noiommu.conf
+        mode: 0644
+        contents:
+          source: data:;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+EOF
+echo "Creating VFIO NoIOMMU MachineConfig"
+oc create -f 05-vfio-noiommu.yaml
+
+git clone https://github.com/rh-nfv-int/shift-on-stack-vhostuser/
+cd shift-on-stack-vhostuser
+NETWORK_ID=$(openstack network show ${VFIONETWORK} -f value -c id)
+ansible-playbook play.yaml -e network_ids="$NETWORK_ID"
```

Comment:
I don't know if ansible is installed (I suspect not), you'll need to install that in the container image.

Reply:
I don't know if we need to load KUBECONFIG or if it's done automagically alraedy.

Reply:
At least I would add:


```
if test ! -f "${KUBECONFIG}"
then
	echo "No kubeconfig, can not continue."
	exit 0
fi
```

Reply:
ansible is installed on the openstack-installer image which is what we'll use to execute this. Now the oc command might be a problem, but in reality we should only craste the yaml files and drop them where required to be used by another step. 
but for now lets use the oc in the openstack-installer just to get this going.  
And it looks like we have to set K8S_AUTH_KUBECONFIG

to the pathc of kubeconfig. 
That is available to the step as ${KUBECONFIG}